### PR TITLE
Guider gains

### PR DESCRIPTION
**Pull request by cquiroz**
This PR adds a parameter to configure guiders to set all gains to 0 as is usual for day time tests

### CODE REVIEW

File: modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala [hunk 1]
```diff
@@ -382,7 +388,7 @@ class TcsBaseControllerEpics[F[_]: Async: Parallel](
 
     config.m2Guide match {
       case M2GuideConfig.M2GuideOff               =>
-        m1(tcsEpics.startCommand(timeout)).m2GuideCommand
+        m1(gains(tcsEpics.startCommand(timeout))).m2GuideCommand
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala:391]: Is this the correct way to chain these calls?

File: modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala [hunk 2]
```diff
@@ -569,11 +569,184 @@ class TcsBaseControllerEpicsSpec extends CatsEffectSuite {
...
+      assertEquals(r1.mountGuide.source.value, "SCS".some)
+
+      assertEquals(r2.m1Guide.value.flatMap(Enumerated[BinaryOnOff].fromTag), BinaryOnOff.Off.some)
+      assertEquals(r2.m2Guide.value.flatMap(Enumerated[BinaryOnOff].fromTag), BinaryOnOff.Off.some)
+      assertEquals(r2.mountGuide.mode.value.flatMap(Enumerated[BinaryOnOff].fromTag),
+                   BinaryOnOff.Off.some
+      )
+      assertEquals(r1.guiderGains.p1TipGain.value, "0.03".some)
+      assertEquals(r1.guiderGains.p1TiltGain.value, "0.03".some)
+      assertEquals(r1.guiderGains.p1FocusGain.value, "2.0E-5".some)
```
Comment (cquiroz) [inline modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala:646]: This is encoded with the default double encoder, will the TCS work fine with numbers encoded this way?

File: modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala [hunk 3]
```diff
@@ -993,6 +1031,30 @@ object TcsEpicsSystem {
       )
     override val oiWfsCmds: WfsCommandsChannels[F]                                       =
       WfsCommandsChannels.build(channels.telltale, channels.oiwfs)
+
+    override val p1GuiderGainsCmd: Command3Channels[F, Double, Double, Double] =
+      Command3Channels(
+        channels.telltale,
```
Comment (jluhrs) [inline modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala:1037]: PWFS1 channels come from a different IOC, they should use a different telltale channel. Same for PWFS2 and OIWFS.
For PWFS1: pwfs1:health.VAL
for PWFS2: pwfs2.health.VAL
for OIWFS: ag:hlth:oiwfs:health.VAL

File: modules/server/src/main/scala/navigate/server/tcs/TcsChannels.scala [hunk 4]
```diff
@@ -493,6 +509,40 @@ object TcsChannels {
...
+        p1FocusGain <- service.getChannel[String]("pwfs1:dc:detSigInitFgGain.C")
+        p1Reset     <- service.getChannel[BinaryYesNo]("pwfs1:dc:initSigInit.J")
+        p2tipGain   <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.A")
+        p2tiltGain  <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.B")
+        p2FocusGain <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.C")
+        p2Reset     <- service.getChannel[BinaryYesNo]("pwfs2:dc:initSigInit.J")
+        oitipGain   <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.A")
+        oitiltGain  <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.B")
+        oiFocusGain <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.C")
+        oiReset     <- service.getChannel[BinaryYesNo]("oiwfs:dc:initSigInit.J")
```
Comment (jluhrs) [inline modules/server/src/main/scala/navigate/server/tcs/TcsChannels.scala:529]: Top should be a parameter for every system: PWFS1, PWFS2, OIWFS. That helps when testing with an EPICS simulator

**Review by jluhrs — CHANGES_REQUESTED** [review]

File: modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala (hunk 1 above)
Comment (jluhrs) [reply modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala:391]: It is prettier if you use `.combine`

File: modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala (hunk 2 above)
Comment (jluhrs) [reply modules/server/src/test/scala/navigate/server/tcs/TcsBaseControllerEpicsSpec.scala:646]: Yes

**Review by jluhrs — APPROVED** [review]
LGTM, but we need to review the telltale channels for o1, p2 and oi at some point.

**Pull request merged**